watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1579,7 +1579,7 @@ def _get_performance_contents(self, time_point=0):
             time_point
         ].flow_vol_phase["Liq"]
 
-        expr_dict["Average Volumetric Flux (LMH)"] = (
+        expr_dict["Average Volumetric Flux"] = (
             self.flux_vol_water_avg[time_point] * 3.6e6
```

Comment:
I believe this needs the same treatment from #1424 (remove the hardcoded *3.6e6)

Reply:
Thanks for catching this!